examples/indexer/src/indexer.ts

```diff
@@ -43,7 +44,11 @@ export function createIndexerConfig(streamUrl: string) {
     },
     sink,
     async transform({ block: { header }, context }) {
-      const { db } = useSink({ context });
+      const { transaction } = useSink({ context });
+
+      assert(header !== undefined);
+
+      const db = transaction({ orderKey: header?.number });
```

Comment:
My idea is that the `useSink` hook should return an instance of `db` already initialized.